### PR TITLE
Panel_7

### PR DESCRIPTION
**Pull request by Jiaqi-beep**
- NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/panel_7.ncl
- NCL Plot: https://www.ncl.ucar.edu/Applications/Images/panel_7_lg.png
- Issue: #334 
hello

### CODE REVIEW

**Comment by anissa111** [comment]
@Jiaqi-beep I think I figured it out! You need a hatch specification for every contour range, not every contour level, which includes above and below the last identified contour level. You actually need 11 hatch specifications, 

For the first:
`hatches=['//', '//', '', '', '', '', '', '', '', '..', '..'],`

For the second:
`hatches=['//', '//', '', '', '', '', '', '', '', '', ''],`

For the third:
`hatches=['', '', '', '', '', '', '', '', '', '..', '..'],`

File: Plots/Panels/NCL_panel_7.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
+"""
+NCL_panel_7.py
+===============
```
Comment (anissa111) [inline Plots/Panels/NCL_panel_7.py:3]:
```suggestion
NCL_panel_7.py
==============
```

File: Plots/Panels/NCL_panel_7.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+This script illustrates the following concepts:
+   - Paneling three plots vertically on a page
+   - Removing tickmarks and labels from paneled plots so they can be drawn closer together
+   - Drawing shaded contours
```
Comment (anissa111) [inline Plots/Panels/NCL_panel_7.py:7]:
```suggestion
   - Drawing shaded contours with hatching
```

**Comment by Jiaqi-beep** [comment]
Oh awesome. Thanks a lot! @anissa111

File: Plots/Panels/NCL_panel_7.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+              zorder=5)
+
+    # Use geocat.viz.util convenience function to add left and right title to the plot axes.
+    gvutil.set_titles_and_labels(axes,
+                                 lefttitle="Zonal Wind",
+                                 lefttitlefontsize=12,
+                                 righttitle=U.units,
+                                 righttitlefontsize=12)
+
+# Panel 1: Contourf-plot U data (for shading)
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_7.py:110]: I think this comment (and the ones for 2 and 3) should be edited to be a bit more specific. Maybe something like:
```suggestion
# Panel 1: Contourf-plot U data with '//' and '..' hatch styles
```
but open to other routes too!

**Review by michaelavs — COMMENTED** [review]
Great start! I pushed one change to add spaces to 'line width' and 'dot size' on 143, and have one comment for this review. I will approve once we settle on if there should be a change or not for my comment/review!

File: Plots/Panels/NCL_panel_7.py (hunk 3 above)
Comment (Jiaqi-beep) [reply Plots/Panels/NCL_panel_7.py]: Thanks for the suggestions. Makes sense to me!

**Review by anissa111 — APPROVED** [review]
This looks good to me, great job!

**Review by michaelavs — APPROVED** [review]
Looks good to me!

**Pull request merged**